app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,12 +1,18 @@
 import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
 import { Button, Pane } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
+import { Input, Textarea, Select } from "neetoui/formik";
 
 import notesApi from "apis/notes";
+import { buildSelectOption } from "utils/index";
```

Comment:
You don't need to specify `/index`. That will be automatically resolved. Read about webpack resolvers.